### PR TITLE
Add alpine-aws container

### PR DESCRIPTION
**Pull request by mbj**
Adds a small container capable to run `aws-cli`.

### Acceptance

Run the following and observe the print out of all S3 buckets.

```
docker run --interactive --tty --volume $HOME/.aws:/root/.aws dkubb/alpine-aws aws s3 ls
````

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

File: alpine-aws/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# dkubb/alpine-aws
+
+FROM alpine:3.6
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+# Upgrade system dependencies
+RUN apk upgrade --update-cache --available
+
+# Install system dependencies
+RUN apk add --update-cache py2-pip=9.0.1-r1
```
Comment (dkubb) [inline alpine-aws/Dockerfile:10]: Since we perform `--update-cache` on the line above, is it necessary to do it again here?

Comment (mbj) [reply alpine-aws/Dockerfile:10]: No, I'll remove it.

**Comment by dkubb** [comment]
@mbj a later PR should add alpine-aws to the `build.sh` script. It appears as if some of the existing containers builds are broken though, so those will need to be fixed first.

File: alpine-aws/Dockerfile (hunk 1 above)
Comment (dkubb) [reply alpine-aws/Dockerfile:10]: @mbj ok, I've merged this, but if you want you can push a new PR with the removal.

**Pull request merged**